torchvision/prototype/models/vgg.py

```diff
@@ -106,6 +106,23 @@ class VGG16Weights(Weights):
             "acc@5": 90.382,
         },
     )
+    # We port the features of a VGG16 backbone trained by amdegroot because unlike the one on TorchVision, it uses the
+    # same input standardization method as the paper. Only the `features` weights have proper values, those on the
+    # `classifier` module are filled with nans.
+    ImageNet1K_Features = WeightEntry(
```

Comment:
We are adding the weights as a separate entry on the VGG. This one only has the features part of the model.